src/openforms/submissions/tasks/co_sign.py

```diff
@@ -30,17 +32,14 @@ def send_email_cosigner(submission_id: int) -> None:
         )
         return
 
-    # Send Co-sign email # TODO add configurable templates
-    content = render_email_template(
-        ugettext(
-            'This is a request to co-sign form "{{ form_name }}". Please go to the webpage of this form on our '
-            "website and click on the 'co-sign' button. You will then be redirected to authenticate yourself."
-            "After authentication, fill in the following code to retrieve the form submission:\n\n{{ code }}\n\n."
-        ),
-        context={
+    template = get_template("emails/cosign.html")
+    content = template.render(
```

Comment:
as remarked by Silvia, I don't think this will actually use the submission language for the context since no translation is activated here. See `openforms.emails.confirmation_emails.get_confirmation_email_templates` for a reference.